DigitalLearningSolutions.Data/Models/CourseContent/CourseContent.cs

```diff
@@ -2,8 +2,26 @@
 {
     public class CourseContent
     {
-        public string CustomisationName { get; set; }
-        public string ApplicationName { get; set; }
-        public int Id { get; set; }
+        public int Id { get; }
+        public string Title { get; }
+        public string AverageDuration { get; }
+        public string CentreName { get; }
+        public string? BannerText { get; }

```

Comment:
Can these have private getters now that you're using a constructor? 🙂

Reply:
I don't think so, because the view model will need to be able to access these fields 🙂 I might be missing something here though

Reply:
Ah sorry, I meant `setter` 🤦

Reply:
Sure! Done in 4ec78e9 🙂